modules/ROOT/pages/vm/vm-reference.adoc

```diff
@@ -5,9 +5,9 @@ endif::[]
 
 Support Category: Select
 
-The VM Connector is used for intra app and inter app communication. The communication is done through asynchronous queues, which can be either transient or persistent. Transient queues are faster, but not reliable in the case of a system crash. Persistent queues, on the other hand are slower but reliable. 
+The VM Connector is used for intra app and inter app communication. The communication is done through asynchronous queues, which can be either transient or persistent. Transient queues are faster, but not reliable in the case of a system crash. Persistent queues, on the other hand are slower but reliable.
 
-When running on a single instance, persistent queues work by serializing and storing the contents on to disk. When running in cluster mode, persistent queues are instead backed by the memory grid. This means that when a flow uses the VM connector to publish content to a queue, the Runtime decides whether to process that message in the same origin node or to send it out to the cluster for another node to pick it up. This is an easy way to distribute load across the cluster. In either way, transactions are always supported. 
+When running on a single instance, persistent queues work by serializing and storing the contents on to disk. When running in cluster mode, persistent queues are instead backed by the memory grid. This means that when a flow uses the VM connector to publish content to a queue, the Runtime decides whether to process that message in the same origin node or to send it out to the cluster for another node to pick it up. This is an easy way to distribute load across the cluster. In either way, transactions are always supported.
```

Comment:
Can't use "Runtime" Have to use Mule or Mule runtime engine

Last line should be loads (I think?)
I think the last sentence is totally unnecessary.